docs/configure-keyperf.md

```diff
@@ -0,0 +1,137 @@
+| - `data`           | duration (Go-syntax)           | poll frequency of updating the data cache <br /><br />**Note** Harvest allows defining poll intervals on sub-second level (e.g. `1ms`), however keep in mind the following:<br /><ul><li>API response of an ONTAP system can take several seconds, so the collector is likely to enter failed state if the poll interval is less than `client_timeout`.</li><li>Small poll intervals will create significant workload on the ONTAP system, as many counters are aggregated on-demand.</li><li>Some metric values become less significant if they are calculated for very short intervals (e.g. latencies)</li></ul> |  1  minute |
+
+The template should define objects in the `objects` section. Example:
+
+```yaml
+objects:
+Aggregate: aggr.yaml
+```
+
+For each object, we define the filename of the object configuration file. The object configuration files are located in subdirectories matching the ONTAP version that was used to create these files. It is possible to have multiple version-subdirectories for multiple ONTAP versions. At runtime, the collector will select the object configuration file that closest matches the version of the target ONTAP system.
+
+### Object Configuration File
+
+The object configuration file should contain the following parameters:
+
+| Parameter        | Type                 | Description                                                 | Default |
+|------------------|----------------------|-------------------------------------------------------------|---------|
+| `name`           | string, **required** | Display name of the collector that will collect this object |         |
+| `query`          | string, **required** | REST endpoint used to issue a REST request                  |         |
+| `object`         | string, **required** | Short name of the object                                    |         |
+| `counters`       | string               | List of counters to collect (see notes below)               |         |
+| `plugins`        | list                 | Plugins and their parameters to run on the collected data   |         |
+| `export_options` | list                 | Parameters to pass to exporters (see notes below)           |         |
+
+#### Template Example
+
+```yaml
+name:                     Volume
+query:                    api/storage/volumes
+object:                   volume
+
+counters:
+- ^^name                                => volume
```

Comment:
Looks like a copy/paste issue. Our templates align the `-` under the `u` in `counters` so this block needs to be shifted to the right by two. Looks to be the case for most sections